### ISSUE TITLE
CreateDerivativesJob is creating temp file with spaces

### ISSUE THREAD

**Issue by carolyncole** [psu-libraries/scholarsphere-3#508]
CreateDerivativesJob is creating a temp file with space which is failing in resque

```
Worker
ssjobs1qa:16823 on files at 2016/12/19 09:37:34 -0500
Retry or Remove
Class
ActiveJob::QueueAdapters::ResqueAdapter::JobWrapper
Arguments
---
job_class: CreateDerivativesJob
job_id: 57e1e60a-04b1-406f-a93e-a2716e40672d
queue_name: files
arguments:
- _aj_globalid: gid://scholar-sphere/FileSet/gf06g2885
- gf06g2885/files/77789ea1-9e7f-4c2a-aa4e-35375e4c9bfd
- "/tmp/gf06g2885_version1"
locale: en
Exception
URI::InvalidURIError
Error
bad URI(is not URI?): /tmp/2013-11-19 Memo for PPE1 interview.pdf
```

**Comment by carolyncole** [comment]
Also we have ones with commas & dashes:
```
bad URI(is not URI?): /tmp/CV, Nelson.pdf
...
bad URI(is not URI?): /tmp/SP2-Jennifer Pressley Revised.pdf
```

**Comment by carolyncole** [comment]
What seems to be happening here is that the file is expected to exists, but it does not.  This is causing the bad URI and  `No such file or directory @ rb_sysopen `

**Comment by carolyncole** [comment]
This error is no longer occurring during the import

**Comment by carolyncole** [comment]
Unfortunately we got one in the last import:

```
---
job_class: ImportVersionJob
job_id: c0d30855-53c4-4259-9413-6456788a8e13
queue_name: files
arguments:
- _aj_globalid: gid://scholar-sphere/FileSet/nvx021f54g
- "/opt/heracles/deploy/scholarsphere/releases/20170217172726/tmp/uploads/nvx021f54g_version1_RFP
  for Montgomery Burns Library System.rtf"
locale: en
Exception
URI::InvalidURIError
Error
bad URI(is not URI?): /tmp/nvx021f54g_version1_RFP for Montgomery Burns Library System.pdf
```

**Comment by carolyncole** [comment]
This also seems to fix #661

**Comment by carolyncole** [comment]
I have rerun several of the failing jobs after the code was deployed to QA.  None of those jobs continued to fail.   Need to run a full import to completely verify.
